src/applications/disability-benefits/all-claims/pages/contactInformation.js

```diff
@@ -55,6 +55,23 @@ const mailingAddress = merge(fullSchema.definitions.address, {
   },
 });
 
+const uiOrder = [
+  'view:livesOnMilitaryBase',
+  'view:livesOnMilitaryBaseInfo',
+  'country',
+  'addressLine1',
+  'addressLine2',
+  'addressLine3',
+  'city',
+  'state',
+  'zipCode',
+];
+// livesOnMilitaryBase is automatically being dropped on
```

Comment:
I assume you mean `livesOnMilitaryBaseInfo` here, since that's what is being dropped from `uiOrder`.

Reply:
Ah yes, good catch, I'll change that.